change-validation/src/main/scala/com/normation/plugins/changevalidation/NotificationService.scala

```diff
@@ -246,7 +264,7 @@ class NotificationService(
       , "description" -> cr.info.description
       , "author"      -> cr.owner
       // rudderBaseUrl should not contains "/" at the end
-      , "link"        -> (rudderBaseUrl + RudderConfig.linkUtil.changeRequestLink(cr.id))
```

Comment:
@ElaadF : you can't use RudderConfig in a service, it's a runtime class. You only have the right to use it when the plugin is init in the context of rudder, so: use a class parameter for services.